test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -81,6 +71,11 @@ public void parse_listCommand_parsedCorrectly() {
         parseAndAssertCommandType(input, ListCommand.class);
     }
 
+    @Test
+    public void parse_luckyCommand_parsedCorrectly() {
```

Comment:
Could include a Junit test to test if the Lucky Command generate a random contact, and if all the contact have a probability of getting picked.

Reply:
Good idea! Will try to implement if I have time.